packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -8,19 +8,13 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart' hide TableRow;
 import 'package:flutter/services.dart';
 
-import '../primitives/auto_dispose_mixin.dart';
+import '../../devtools_app.dart';
```

Comment:
import libraries directly instead of importing `devtools_app.dart`, which exports a bunch of libraries in devtools